src/DatePicker/datePickerItem.css

```diff
@@ -50,10 +50,10 @@
 
 .type__day
 {
-    height:             calc( var( --spacing-4 ) - 2px );
+    height:             calc( var( --spacing-4 ) );
```

Comment:
Looks nice! One small thing here: since there's nothing to calculate, this should just be `var( --spacing-4 )`.

Reply:
Done 👍.